tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py35,py36,py37,py38,py39,pypy2
```

Comment:
Are we losing test coverage for pypy?

Reply:
Is it better to add `pypy35` or something to test with pypy3X?
https://github.com/tox-dev/tox/blob/69af6cdf168df4e82d6ce0f3d05571bd100418f9/docs/example/basic.rst

Reply:
It would surely be good to do that in this PR itself if the tests are passing.

If CI fails on pypy3x let's create an issue for the follow-up.

Reply:
Looks like we aren't using tox in CI and instead directly run pytest `https://github.com/trinodb/trino-python-client/blob/master/.github/workflows/ci.yml#L31`.

Let's do this in a follow-up since this looks pre-existing anyway.

cc: @joshthoward

Reply:
Hmm... reverted.